app/views/spree/admin/products/new.html.erb

```diff
@@ -1,3 +1,8 @@
+<script>
+  $('.flatpickr-alt-input').not(':first-child').remove()
```

Comment:
@aleksandarpetrushev this does not resolve this issue, the issue occurs because some events/methods are attached every time someone traverses the page 

* https://turbo.hotwired.dev/handbook/drive
* https://turbo.hotwired.dev/reference/drive

The problem is that we have a script block here - https://github.com/spree/spree_backend/blob/main/app/views/spree/admin/products/new.html.erb#L85

This will not work with Turbo, this needs to be rewritten like the rest of the select2 pickers

Reply:
@damianlegawiec commenting out that script does not fix it apparently. How should it be rewritten, I can not seem to find the difference between this and the other select2 pickers

Reply:
it happens on the menus page as well